index.html

```diff
@@ -9,7 +9,45 @@
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Climate+Crisis&family=Poppins:ital,wght@0,400;0,600;0,700;0,800;0,900;1,400&display=swap" rel="stylesheet">
 </head>
-
+<style>
+  .reviews {
+    text-align: center;
+    padding: 20px;
+    font-family: 'Poppins';
+  }
+  #review-list{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    margin-top: 20px;
+  }
+
+  .review-items{
+    display: flex;
+    align-items: center;
+    gap: 30px;
+    padding:20px;
+    width: 80%;
+    max-width: 500px;
+    border-radius: 20px;
+    box-shadow:0 4px 6px rgba(0, 0, 0, 0.2);
+    background-color: #fff;
+    
+  }
+  .review-items:hover{
+    transform: scale(1.1);
+    box-shadow: 0 8px 15px rgba(0, 0, 0, 0.4);
+  }
+  .profile-pic{
+    width: 60px;
+    height: 60px;
+    border-radius: 50%;
+  }
+  .review-content{
+    text-align: left;
+  }
+</style>

```

Comment:
Please move this to its corresponding CSS file

Reply:
This is still not fixed, please remove it from the index.html file to CSS files/recipepage.css